RealEstate/apps/core/templates/core/categories.html

```diff
@@ -85,11 +45,11 @@
 	{% endfor %}
 </ul>
 <center>
-    <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#addCategory" style="margin:30px">Add Category</button>
+    <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#addCategory" id="button-margin">Add Category</button>
```

Comment:
Yeah, I think we can leave out the button-margin here. Let's style all the buttons the same and call it by its element name: ```button {  }```